cmd/s3-gw/app_settings.go

```diff
@@ -32,8 +32,8 @@ const ( // Settings.
 	cfgLoggerLevel = "logger.level"
 
 	// Wallet.
-	cfgWallet           = "wallet"
-	cfgAddress          = "address"
+	cfgWalletPath       = "wallet.path"
```

Comment:
Can we keep flag named `wallet`? And map it to `wallet.path` viper config value?

Reply:
Here is the example how we bind flag name with viper config value: https://github.com/nspcc-dev/neofs-s3-gw/blob/e5c1acf1e59d7ff9132f7bb5f059687e72ffce59/cmd/s3-gw/app_settings.go#L217-L226